HA/xcatha.py

```diff
@@ -67,20 +67,20 @@ def run_command(cmd, retry, ignore_fail=None):
     global dryrun
     if dryrun:
         loginfo=cmd+" [Dryrun]"
```

Comment:
Why do we need to append this `Dryrun` to the output?

Reply:
This is so that when we display something like:
`ifconfig eth0:0 0.0.0.0 0.0.0.0 &>/dev/null [Dryrun]`
it is obvious that the command was not really executed.

Reply:
Since logger has no print "DRYRUN", I think `[Dryrun]` looks obvious behind each command.